### ISSUE TITLE
Package request: gpsd

### ISSUE THREAD

**Issue by JGrana01** [Entware-ng/Entware-ng#71]
The present Entware-ng has ntpd. It would be helpful to also have gpsd (and cgps) added. This would allow USB based GPS devices to be used as part of the NTP calculation.

**Comment by ryzhovau** [comment]
Done. https://github.com/Entware-ng/entware-packages/commit/9044cb1603bde5ffb6cb999e6212e5410e03e8a4 ARM/x86 binaries will arrive in a couple of days.

I saw [those thread](http://www.snbforums.com/threads/ntp-daemon-for-asuswrt-merlin.28041/), it's impressive. Just curious: are you going to put router on window to receive GPS?

**Comment by zyxmon** [comment]
ARM/x86 binaries are already in repo )))

**Comment by JGrana01** [comment]
Thanks guys! it works great. ryzhovau, I have for now an old USB GPS tracker I used to use for geotagging my photos. It is pretty sensitive and I am getting 8 satellites in the upper floor where the router is located.

I have installed both your ntpd and gpsd. I then put in kvics fancy web tab for Asuswrt-merlin. Working well!

My next project is to interface to a used Garmin GPS 18X LVC I purchased on ebay. I am going to attempt to get PPS precision.
